### ISSUE TITLE
Integration fail, can´t connect

### ISSUE THREAD

**Issue by SirMartin** [Farfar/homeassistant-seatconnect#33]
Hi,

Like a week ago, started the integration to fail, and cannot import **Connection**

```
Logger: homeassistant.setup
Source: setup.py:214
First occurred: May 29, 2023 at 06:12:52 (1 occurrences)
Last logged: May 29, 2023 at 06:12:52

Setup failed for custom integration seatconnect: Unable to import component: cannot import name 'Connection' from 'seatconnect' (/usr/local/lib/python3.10/site-packages/seatconnect/__init__.py)
```

Also, I tried to remove the integration and create a new one but got almost the same fail.

```
Logger: homeassistant.config_entries
Source: config_entries.py:863
First occurred: 18:57:50 (1 occurrences)
Last logged: 18:57:50

Error occurred loading flow for integration seatconnect: cannot import name 'Connection' from 'seatconnect' (/usr/local/lib/python3.10/site-packages/seatconnect/__init__.py)
```

Any idea?

I appreciate any help you can provide.

**Comment by SirMartin** [comment]
I am the only one with the issue? I finally can remove the integration, then I re-downloaded in HACS, but when I try to add the integration the config flow fails, because of the same issue.

```
Logger: homeassistant.config_entries
Source: config_entries.py:863
First occurred: 15:45:43 (1 occurrences)
Last logged: 15:45:43

Error occurred loading flow for integration seatconnect: cannot import name 'Connection' from 'seatconnect' (/usr/local/lib/python3.10/site-packages/seatconnect/__init__.py)
```

I have the latest version of Home Assistant right now.

```
Home Assistant 2023.5.4
Supervisor 2023.04.1
Operating System 10.2
Frontend 20230503.3 - latest
```

If I have time I will try to make in a virtual machine a new installation and see if happens again, and I will try to download the library and check if locally without Home Assistant works the connection.

**Comment by SirMartin** [comment]
It was my problem, and finally I fixed.

Somehow, the seatconnect library package was corrupted (probably after an hass update), and stop to work. I created a VM and tested and worked correctly, so I knew it was not a integration issue, just for my installation. I tried to uninstall, remove from HACS, and everything, but when I installed again (after reboots, etc), I got the same error, so I knew something happened with the package installed in site packages.

Finally through a SSH I was able to enter to the "homeassistant" docker container, look for the folder, remove all the related content to this integration, uninstall everything from HACS related to this plugin, reboot the machine, and after it just try to install it normally, and it is working correctly again.

I write this here, so maybe it is helpful for someone in the future.
